apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -430,11 +433,13 @@ const FunctionsOrEventsListItem: React.FC<FunctionsOrEventsListItemProps> = ({
   const pathname = usePathname();
   const router = useDashboardRouter();
   return (
-    <ListItem my={0.5}>
+    <ListItem my={1}>
       <Button
         size="sm"
-        fontWeight={isActive ? 600 : 400}
-        opacity={isActive ? 1 : 0.65}
+        className={cn(
+          "!font-medium !text-muted-foreground hover:!text-foreground font-mono",
+          isActive && "!text-foreground",
+        )}
```

Comment:
prefer the object notation for the conditional className


```suggestion
        className={cn("!font-medium !text-muted-foreground hover:!text-foreground font-mono", {
          "!text-foreground": isActive
        })}
```

Reply:
Sure, Updated it